### DISCUSSION TITLE
Webpack detecting changes even when no change in the files content

### DISCUSSION THREAD

**Discussion by leimonio** [webpack/webpack#17710] [General]
I'm using webpack and typescript without `ts-loader`. Typescript compilation triggers as a separate task outside webpack.
When I trigger a change in a file in my codebase, the typescript transpilation will get triggered and produce transpiled files in the which is part of the entry point tree webpack is registered to. 

The problem is that webpack will getting triggered and reprocess the files, and thus retrigger the dev server even when no content changes have been made in the actual file.

I've tried to use the `snapshot.module` by disabling the timestamp of the file to be counted as a change in the file without any luck. https://webpack.js.org/configuration/other-options/#module

I can still see that webpack is getting triggered and processing all the files.

```
module: {
        timestamp: false,
        hash: true,
},
```

**Comment by alexander-akait** [comment]
Sorry for delay, is it still valid? If you please create reproducible test repo, thank you
